package.json

```diff
@@ -2,12 +2,12 @@
   "name": "platformicons",
   "version": "5.5.0",
   "description": "Web Platform and Framework Logo Set",
-  "main": "build/index.js",
-  "types": "build/index.d.ts",
+  "sideEffects": false,
+  "main": "build/cjs/index.js",
+  "module": "build/esm/index.js",
+  "types": "build/cjs/index.d.ts",
```

Comment:
Will deployment tooling need to change with the different build directories?

Reply:
all bundlers should know what to do with this. i'll still make this a breaking change and bump the major version.

listing a main and a module export is the first step towards esm and lets the bundler use either export